src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFandomUap.java

```diff
@@ -84,10 +84,9 @@ private void verifyUapAtf(long atfId, String slotName, AdsFandomObject fandomPag
 
   private void verifyUapBtf(long btfId, String slotName, String bottomSlotName, AdsFandomObject fandomPage) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "verifyUapBtf" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)